phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexToolIT.java

```diff
@@ -617,6 +618,125 @@ public void testSplitIndex() throws Exception {
         }
     }
 
+    @Test
```

Comment:
These tests should move into IndexToolForNonTxGlobalIndexIT after the recent restructuring of IndexToolIT